controllers/release/adapter_test.go

```diff
@@ -872,14 +872,26 @@ var _ = Describe("Release adapter", Ordered, func() {
 		})
 
 		It("references the pipeline specified in the ReleasePlanAdmission", func() {
-			var pipelineName string
+			var pipelineUrl string
 			resolverParams := pipelineRun.Spec.PipelineRef.ResolverRef.Params
 			for i := range resolverParams {
-				if resolverParams[i].Name == "name" {
-					pipelineName = resolverParams[i].Value.StringVal
+				if resolverParams[i].Name == "url" {
+					pipelineUrl = resolverParams[i].Value.StringVal
 				}
 			}
-			Expect(pipelineName).To(Equal(releasePlanAdmission.Spec.PipelineRef.Params[1].Value))
+			Expect(pipelineUrl).To(Equal(releasePlanAdmission.Spec.PipelineRef.Params[0].Value))
+		})
+
+		It("contains a parameter with the taskGitRevision", func() {
+			Expect(pipelineRun.Spec.Params).Should(ContainElement(HaveField("Name", "taskGitRevision")))
+			var revision string
+			resolverParams := pipelineRun.Spec.PipelineRef.ResolverRef.Params
+			for i := range resolverParams {
+				if resolverParams[i].Name == "revision" {
```

Comment:
I was trying to figure out if the pipelinerun will actually have it. Can you help me find where it's defined? The ppr is created in the `BeforeEach` function, right? And I guess I need to look at the `releasePlanAdmission` referenced in resources. But where is that one defined? Is `loader_mock` involved here?

Reply:
Oh, now I see it - `createResources()` is called in the `BeforeAll` on line 73. So that's where it appears. Thanks for the explanation.